### DISCUSSION TITLE
Can it install under Kodi 20 Nexus?

### DISCUSSION THREAD

**Discussion by jcjdammers** [Goldenfreddy0703/plugin.audio.ytmusic.exp#25] [Q&A] [answered]
My question is:

Can this addon also be installed under Kodi 20 Nexus?
I am using LibreELEC 11.0.3.
Thanks for your answer,
Jeroen

**Comment by Goldenfreddy0703** [comment, marked as answer]
Hey so sadly I'm not too sure, I think so but I haven't worked on youtube music in a long time and the code feels disorganized and all sadly. We need more contributors on this but we also need to do a recode at some point.

**Comment by fischcode** [comment]
I have plugin.audio.ytmusic.exp running under Kodi 20 Nexus on LibreElec (Raspberry Pi 4) and Windows 11 without problems. Give it a try!

**Reply by wyup** [reply]
Can it act as a renderer for android YouTube music app?
